File: year2020/month09/day0915/ms_0915.cpp
#include <iostream>

using namespace std;

int Square(int x, int y);

int main() {
	int data[2] = { 0, 0 };

	for (int i = 0; i < 2; i++) {
		int temp;

		scanf_s("%d", &temp);

		if (temp >= 0)
			data[i] = temp;
	}

	cout << Square(data[0], data[1]) << endl;

	return 0;
}

int Square(int x, int y) {
	if (x > y)
		return x * x - y * y;
	else
		return y * y - x * x;
}

File: year2020/month09/day0915/ms_0915.cs
using System;

namespace algorithm_57
{
    class ms_0915
    {
        static int Square(int x, int y)
        { 
            if(x > y)
                return x*x - y*y;
            else
                return y*y - x*x;
        }

        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' ');
            int[] data = new int[2] { 0, 0 };

            for(int i = 0; i < data.Length; i++) 
            {
                int temp = Convert.ToInt32(input[i]);

                if(temp >= 0)
                    data[i] = temp;
            }

            Console.WriteLine(Square(data[0], data[1]));
        }
    }
}
